test/Microsoft.Extensions.Logging.Test/LoggerExtensionsTest.cs

```diff
@@ -38,10 +38,15 @@ public void MessageOnly_LogsCorrectValues()
             logger.LogError(_state);
             logger.LogCritical(_state);
             logger.LogDebug(_state);
+            logger.Log(LogLevel.Trace, _state);
```

Comment:
Add a separate test method that calls new overloads (with exception and non zero event id)

Reply:
Sorry for the delay.  I'll get to this now.

Reply:
I've added separate tests for the new overloads, one for each of the 10 "flavors" of the equivalent `logger.LogLEVELNAME()` extensions.  Each uses a `LogLevel_` prefix to the test method name ([Example](https://github.com/aspnet/Logging/pull/790/files#diff-099caadefe774232f5d421983c3ee853R627)).  Let me know if that naming works.

(FYI, I rebased and squashed commits, @pakrym.)